projects/plugins/jetpack/extensions/plugins/publicize/components/connection-toggle/style.scss

```diff
@@ -0,0 +1,17 @@
+.components-connection-toggle {
+	position: relative;
+	display: flex;
+	align-items: center;
+	width: 100%;
+
+	// Unchecked state.
+	&.is-not-checked .jetpack-gutenberg-social-icon {
+		fill: #dadada;
+	}
+
+	// Disabled state:
+	&.is-disabled,
+	.components-disabled & {
+		opacity: 0.3;
```

Comment:
I think it's too transparent.

<img src="https://user-images.githubusercontent.com/77539/136092827-1c420300-5800-4ff6-a910-b1888b03e118.png" width="300px" />

Reply:
I bumped up the opacity slightly, let me know how it looks:
<img width="260" alt="Screen Shot 2021-10-06 at 9 44 25 AM" src="https://user-images.githubusercontent.com/1689238/136216662-c6c80571-ca0f-4d06-8316-5bc3fb12122c.png">